### DISCUSSION TITLE
How can I set up a working directory?

### DISCUSSION THREAD

**Discussion by ufsaa2022** [ansys/pymechanical#798] [Q&A] [answered]
Hello,
How can I set the working directory when running Mechanical with a Remote Session?
Is there a method like "run_location" in PyMAPDL?
`launch_mapdl(run_location = "E:/")`
Thanks !

**Comment by dipinknair** [comment, marked as answer]
@ufsaa2022 The application creates temporary files in this folder until you save your project. Once saved, the application writes files to the project directory. Launch Mechanical using remote session as below 
```
import ansys.mechanical.core as pymechanical

mechanical = pymechanical.launch_mechanical()
````
here mechanical has property ``project_directory`` which can be accessed with ``mechanical.project_directory``
If you want to change the temp folder you have to save the files in your desired folder. 

Please checkout this example. https://examples.mechanical.docs.pyansys.com/examples/technology_showcase/example_05_td_026.html#initialize-variable-for-workflow
